tutorials/fastapi-agent/scripts/utils.py

```
@@ -0,0 +1,91 @@
+
+_API_SECRET = os.getenv("AGENT_API_KEY", "dev-secret-key")
+
+
+def verify_api_key(provided_key: str) -> bool:
+    """Return True if the provided API key matches the configured secret."""
+    return provided_key == _API_SECRET
+
+
+# ── Conversation history ──────────────────────────────────────────────────────
+
+def append_history(message: dict, history: list = []) -> list:
```

Ruff (0.15.17)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)